spec/admin.mediawiki

```diff
@@ -32,13 +32,21 @@ possible.
 {|
 ! variable !! units !! description
 |-
-| lot&nbsp;size  || atoms ||  the minimum order quantity and the order quantity increment when an asset is the base asset
+| lot&nbsp;size  || atoms or gwei ||  the minimum order quantity and the order quantity increment when an asset is the base asset
 |-
-| rate&nbsp;step || atoms || the minimum price rate and the price rate increment when an asset is the quote asset. [[orders.mediawiki/#Rate_Encoding|message-rate encoding]]
+| rate&nbsp;step || atoms or gwei || the minimum price rate and the price rate increment when an asset is the quote asset. [[comm.mediawiki/#Rate_Encoding|message-rate encoding]]
 |-
-| fee&nbsp;rate  || atoms/byte || the minimum fee rate for swap transactions
+| fee&nbsp;rate  || atoms/byte or gwei/gas || the minimum fee rate for swap transactions
```

Comment:
Am I interpreting this section wrong? Should these variables mirror this?
```
// Asset is the configurable asset variables.
type Asset struct {
	ID           uint32 `json:"id"`
	Symbol       string `json:"symbol"`
	Version      uint32 `json:"version"`
	LotSize      uint64 `json:"lotSize"`
	RateStep     uint64 `json:"rateStep"`
	MaxFeeRate   uint64 `json:"maxFeeRate"`
	SwapSize     uint64 `json:"swapSize"`
	SwapSizeBase uint64 `json:"swapSizeBase"`
	SwapConf     uint32 `json:"swapConf"`
}
```

Reply:
Hmm, yeah this section of the spec does seem to describe the config items in markets.json (for Per-asset Variables) and dcrdex.conf (for Exchange Variables).

I'm noticing an error with `bip44symbol` being called an `int`.

Regarding this "fee rate", this is another out of date item because it's really *max* fee rate, not minimum.  I do recall we had a different idea about how fees would be set and enforced before we realized it needed to be dynamic based on network conditions with just a max.

Also, I'm thinking we should not set the units of fee rate explicitly since we can't keep adding units to the spec as we add assets.  It'll just have to be described as an integer, with units value per some cost such as bytes or gas.

Reply:
Took out `fee rate`. Made variable names mirror whats in the `markets.json` file. Use in instead of unit names. Fix `bip44symbol` type.